src/extension.ts

```diff
@@ -69,9 +71,51 @@ function npmCheck() {
   });
 }
 
+function getPackageManagerConfig(): PackageManagerConfiguration {
+  const config = vscode.workspace.getConfiguration().inspect('npm.packageManager');
+  const globalConfig = config?.globalValue;
+  const workspaceConfig = config?.workspaceValue;
+
+  const pickValueFromConfig = (config: unknown): PackageManagerConfiguration => {
+    if (typeof config !== 'string' || (config !== 'yarn' && config !== 'npm')) {
+      return undefined;
+    }
+
+    return config;
+  };
+
+  return pickValueFromConfig(workspaceConfig) || pickValueFromConfig(globalConfig);
```

Comment:
The workspace configuration takes priority on the global configuration

Reply:
great 👍